Python/_warnings.c

```diff
@@ -15,6 +15,109 @@ _Py_IDENTIFIER(default);
 _Py_IDENTIFIER(ignore);
 #endif
 
+
+/*************************************************************************/
+
+typedef struct warnings_state {
```

Comment:
Why is this so inconsistent with everything else? (`WarningsState`, `struct warningsmodule`, etc.)

Reply:
fixed